### ISSUE TITLE
Include a short summary of enabled components as part of `bash-it doctor`

### ISSUE THREAD

**Issue by NoahGorny** [Bash-it/bash-it#1745]
Maybe we should include a short summary of enabled components as part of `bash-it doctor`. So instead of having to run several commands, we just run one command to produce the required output...

The list of components could be condensed to only show the enabled ones on one line (one for aliases, one for completions, one for plugins), instead of the line-by-line output that `bash-it show` currently produces.

_Originally posted by @nwinkler in https://github.com/Bash-it/bash-it/pull/1740#discussion_r543165247_

**Comment by cornfeedhobo** [comment]
Yeah, I kinda like the idea of overloading doctor to push people to use it more and get ahead of any misconfigurations.
